### PR TITLE
Drops whole measurement list if some of value is negative

### PR DESCRIPTION
**Pull request by isaikevych**
Issue [206](https://github.com/census-instrumentation/opencensus-node/issues/206)

### CODE REVIEW

File: packages/opencensus-core/src/stats/stats.ts [hunk 1]
```diff
@@ -114,18 +114,26 @@ export class Stats {
     return {name, unit, type: MeasureType.INT64, description};
   }
 
+  /**
+   * Virifies whether all measurements has positive value
```
Comment (mayurkale22) [inline packages/opencensus-core/src/stats/stats.ts]: s/Virifies/Verifies?

File: packages/opencensus-core/src/stats/stats.ts [hunk 2]
```diff
@@ -114,18 +114,26 @@ export class Stats {
...
+   * @param measurements A list of measurements
+   */
```
Comment (mayurkale22) [inline packages/opencensus-core/src/stats/stats.ts]: ```@return``` is missing.

File: packages/opencensus-core/src/stats/stats.ts (hunk 1 above)
Comment (isaikevych) [reply packages/opencensus-core/src/stats/stats.ts]: Done

File: packages/opencensus-core/src/stats/stats.ts (hunk 2 above)
Comment (isaikevych) [reply packages/opencensus-core/src/stats/stats.ts]: Done

**Comment by isaikevych** [comment]
@justindsmith Done

**Pull request merged**